docs/reference/api/platform/Error.md

```diff
@@ -1,5 +1,21 @@
 ## Error
 
-The error function generates a fatal runtime error, which allows the user to dump a message to the serial line, halt the program and place the device in an idle mode with a blinking red LED.
+Mbed OS provides an error function to output messages to `STDIO` at runtime when the system encounters a fatal error and causes the application to terminate. `mbed_error.h` declares this error message. Note that the error function outputs the error message in debug and develop builds only. In release builds, the error function does not generate any output, but the application is still be terminated.
 
-You can read more about the Error function in [its documentation](https://github.com/ARMmbed/mbed-os/blob/master/platform/mbed_error.h).
+### Error function reference
+
+[![View code](https://www.mbed.com/embed/?type=library)](https://os.mbed.com/docs/v5.6/mbed-os-api-doxy/group__platform__error.html)
```

Comment:
Query: Does this link work for you? I see "Page error".

Reply:
Again, it depends on my other change coming into mbed-os side.

Reply:
Awesome. In the description of this PR, could you please list the mbed-os PR#s as dependencies?